RNTester/package.json

```diff
@@ -3,9 +3,10 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "build": "bsb -clean-world -make-world",
+    "build": "node ../node_modules/.bin/bsb -make-world",
+    "clean-build": "node ../node_modules/.bin/bsb -clean-world -make-world",
     "packager:start": "node node_modules/react-native/local-cli/cli.js start",
-    "watch": "bsb -make-world -w",
+    "watch": "node ../node_modules/.bin/bsb -make-world -w",
```

Comment:
what is this?

Reply:
ok, in the local environment `bsb` is usually is installed globally (according to this: https://reasonml.github.io/docs/en/quickstart-javascript.html), and here we must use `bsb` installed in the root `node_modules` (`bsb` belongs to the `bs-platform` package)

Reply:
I get that but do we need to invoke node instead of just the path itself?

Reply:
👍 fixed!